examples/ecdsa_utils/ecdsa_utils.h

```diff
@@ -47,6 +47,21 @@ extern "C" {
 /** @brief Maximum overhead of the ASN.1 encoding for the R and S components of an ECDSA signature */
 #define ECDSA_RS_MAX_ASN1_OVERHEAD ((2 + 1) * 2)
 
+/**
+ * @brief Encodes the ECDSA signature components (r, s) in as two ASN.1 INTEGERs concatenated. This is the format
+ *        the OPTIGA hostcode expects.
+ *
+ * @param[in]   r            Component r of the ECDSA signature
+ * @param[in]   s            Component s of the ECDSA signature
+ * @param[in]   rs_len       Length of the buffers for the R and S components of the ECDSA signature, must be smaller than 127

```

Comment:
"Total length of both buffers" or "length of each buffer" would be less ambigous